internal/versionchecker/test/testdata/fetch.go

```diff
@@ -375,44 +376,44 @@ func cleanupManifests(manifests []byte, version string) ([]byte, error) {
 
 	decoder := yaml.NewDecoder(bytes.NewBuffer(manifests))
 	for {
-		var spec map[string]interface{}
+		var manifest map[string]interface{}
```

Comment:
comment (no action needed): I'm surprised there isn't a linter which suggests changing `interface{}` to `any`; I think it would provide a lot of benefit in this file. Wonder if there's a contribution opportunity there 🤔 Obviously unrelated to this PR though!